examples/dapp/near/wallet-selector-vanilla/assets/global.css

```
@@ -0,0 +1,1820 @@
+  max-width: 812px;
+  height: 555px;
+  border-radius: 16px;
+  position: absolute;
+  left: 50%;
+  transform: translate(-50%, 0px);
+  transition: visibility 0s linear 0s, opacity 0.25s 0s, transform 0.25s;
+  background-color: var(--wallet-selector-content-bg, var(--content-bg));
+  overflow-y: auto;
+  font-size: 16px;
+  line-height: 1.6;
+  overflow: hidden;
```

Biome (2.1.2)

[error] Unexpected shorthand property overflow after overflow-y



(lint/suspicious/noShorthandPropertyOverrides)